server/api/student/index.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+var express = require('express');
+var controller = require('./student.controller');
+
+var router = express.Router();
+
```

Comment:
Let's leave a TODO comment here to lock down these routes with authorization role checks.